examples/launch_sklearn_surrogate_bo.py

```diff
@@ -12,118 +12,111 @@
-        """
-        Returns signals which are statistics of the predictive distribution at
-        input points ``inputs``.
-
-
-        :param inputs: Input points, shape ``(n, d)``
-        :return: Tuple with the following entries:
-            * "mean": Predictive means in shape of ``(n,)``
-            * "std": Predictive stddevs, shape ``(n,)``
-        """
-        print(f"Predicting with BayesianRidgePredictor using X.shape={X.shape}")
+        self, X: np.ndarray
+    ) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
         return self.ridge.predict(X, return_std=True)
 
 
 class BayesianRidgeEstimator(SKLearnEstimator):
     """
-    Base class for the sklearn Estimators
+    Estimator for surrogate model given by ``sklearn.linear_model.BayesianRidge``.
+
+    None of the parameters of ``BayesianRidge`` are exposed here, so they are all
+    fixed up front.
     """
 
     def __init__(self, *args, **kwargs):
         self.ridge = BayesianRidge(*args, **kwargs)
 
     def fit(
         self, X: np.ndarray, y: np.ndarray, update_params: bool
-    ) -> BayesianRidgePredictor:
-        """
-        Implements :meth:`fit_from_state`, given transformed data.
-
-        :param X: Training data in ndarray of shape (n_samples, n_features)
-        :param y: Target values in ndarray of shape (n_samples,)
-        :param update_params: Should model (hyper)parameters be updated?
-        :return: Predictor, wrapping the posterior state
-        """
-        print(
-            f"Fitting BayesianRidgePredictor using X.shape={X.shape}, y.shape={y.shape}"
-        )
+    ) -> SKLearnPredictor:
         self.ridge.fit(X, y.ravel())
         return BayesianRidgePredictor(ridge=copy.deepcopy(self.ridge))
 
 
-def main():
-    # Hyperparameter configuration space
+if __name__ == "__main__":
+    logging.getLogger().setLevel(logging.INFO)
+
+    random_seed = 31415927
+    max_epochs = 100
+    n_workers = 4
+
     config_space = {
         "width": randint(1, 20),
         "height": randint(1, 20),
-        "epochs": 10,
+        MAX_RESOURCE_ATTR: 100,
     }
-    # Scheduler (i.e., HPO algorithm)
-    myestimator = BayesianRidgeEstimator()
+    entry_point = str(
+        Path(__file__).parent
+        / "training_scripts"
+        / "height_example"
+        / "train_height.py"
+    )
+
+    # We use ``FIFOScheduler`` with a specific searcher based on our surrogate
+    # model
     searcher = SKLearnSurrogateSearcher(
         config_space=config_space,
-        metric="mean_loss",
-        estimator=myestimator,
+        metric=METRIC_ATTR,
+        estimator=BayesianRidgeEstimator(),
         scoring_class_and_args=EIAcquisitionFunction,
     )
-
     scheduler = FIFOScheduler(
         config_space,
-        metric="mean_loss",
-        mode="min",
+        metric=METRIC_ATTR,
+        mode=METRIC_MODE,
+        max_resource_attr=MAX_RESOURCE_ATTR,
         searcher=searcher,
-        search_options={"debug_log": False},
```

Comment:
If you pass an object for `searcher`, there is no `search_options`

Reply:
OK, makes sense